### ISSUE TITLE
Not seeing ICMP replies from the blaster when subscriber IP is pinged

### ISSUE THREAD

**Issue by ArunPrakashThangaraj** [rtbrick/bngblaster#240]
**Not seeing ICMP replies from the blaster when subscriber IP is pinged**

**In the latest blaster version, after establishing IPoE subscribers when the subscriber IP is pinged from RBFS, a lot of requests were not honoured with ICMP replies.**

Version (`bngblaster -v`):

```
supervisor@SN-STD-27-192083107:~ $ bngblaster -v
Version: 0.8.36
Compiler: GNU (11.4.0)
IO Modes: packet_mmap_raw (default), packet_mmap, raw
supervisor@SN-STD-27-192083107:~ $ 
```

JSON configuration:
```json
{
{
  "interfaces": {
    "tx-interval": 1,
    "rx-interval": 1,
    "lag": [
      {
        "interface": "lag1",
        "lacp": true,
        "lacp-timeout-short": true
      }
    ],
    "links": [
      {
        "interface": "SN-5-C1",
        "lag-interface": "lag1"
      },
      {
        "interface": "SN-6-C1",
        "lag-interface": "lag1"
      },
      {
        "interface": "SN-7-C1",
        "lag-interface": "lag1"
      },
      {
        "interface": "SN-11-C2",
        "lag-interface": "lag1"
      },
      {
        "interface": "SN-12-C2",
        "lag-interface": "lag1"
      },
      {
        "interface": "SN-13-C2",
        "lag-interface": "lag1"
      }
    ],
    "network": [
      {
        "interface": "SN-15-R1",
        "address": "36.1.1.2/24",
        "gateway": "36.1.1.1",
        "isis-instance-id": 1,
        "isis-level": 1
      },
      {
        "interface": "SN-17-R2",
        "address": "46.1.1.2/24",
        "gateway": "46.1.1.1",
        "isis-instance-id": 2,
        "isis-level": 1
      },
      {
        "interface": "SN-19-RR",
        "address": "131.0.0.2/24",
        "gateway": "131.0.0.1",
        "address-ipv6": "fc66:1337:7331::2",
        "gateway-ipv6": "fc66:1337:7331::1",
        "isis-instance-id": 3,
        "isis-level": 1
      }
    ],
    "access": [
      {
        "interface": "lag1",
        "type": "ipoe",
        "outer-vlan-min": 1001,
        "outer-vlan-max": 1100,
        "inner-vlan-min": 1001,
        "inner-vlan-max": 1100
      }
    ]
  },
  "sessions": {
    "count": 10,
    "max-outstanding": 800,
    "start-rate": 400,
    "stop-rate": 100
  },
  "access-line": {
    "agent-remote-id": "DEU.RTBRICK.{session-global}",
    "agent-circuit-id": "0.0.0.0/0.0.0.0 eth 0:{session-global}",
    "rate-up": 2000,
    "rate-down": 16384,
    "dsl-type": 5
  },
  "dhcp": {
    "enable": true
  },
  "dhcpv6": {
    "enable": true
  },
  "session-traffic": {
    "ipv4-pps": 10,
    "ipv6-pps": 10,
    "ipv6pd-pps": 10,
    "autostart": true
  },
  "isis": [
    {
      "instance-id": 1,
      "area": [
        "49.0002/24"
      ],
      "system-id": "0204.0000.0001",
      "router-id": "192.1.0.36",
      "hostname": "BBL-LSR1",
      "hello-padding": true,
      "teardown-time": 30,
      "external": {
        "mrt-file": "/home/supervisor/isis.mrt",
        "connections": [
          {
            "system-id": "1921.6800.1003",
            "l1-metric": 10
          }
        ]
      }
    },
    {
      "instance-id": 2,
      "area": [
        "49.0002/24"
      ],
      "system-id": "0204.0000.0002",
      "router-id": "192.1.0.46",
      "hostname": "BBL-LSR2",
      "hello-padding": true,
      "teardown-time": 30,
      "external": {
        "mrt-file": "/home/supervisor/isis.mrt",
        "connections": [
          {
            "system-id": "1921.6800.1004",
            "l1-metric": 10
          }
        ]
      }
    },
    {
      "instance-id": 3,
      "area": [
        "49.0002/24"
      ],
      "system-id": "0204.0000.0003",
      "router-id": "192.1.0.56",
      "hostname": "BBL-RR",
      "hello-padding": true,
      "teardown-time": 30,
      "sr-base": 1000,
      "sr-range": 100,
      "sr-node-sid": 56,
      "external": {
        "mrt-file": "/home/supervisor/isis.mrt",
        "connections": [
          {
            "system-id": "1921.6800.1005",
            "l1-metric": 10
          }
        ]
      }
    }
  ],
  "bgp": [
    {
      "__comment__": "RR-IPv4",
      "network-interface": "SN-19-RR",
      "local-ipv4-address": "192.1.0.56",
      "peer-ipv4-address": "192.1.0.51",
      "raw-update-file": "/home/supervisor/ipv4_nlri.bgp",
      "local-as": 3330,
      "peer-as": 3320
    },
    {
      "__comment__": "RR-IPv6",
      "network-interface": "SN-19-RR",
      "local-ipv4-address": "192.1.0.56",
      "peer-ipv4-address": "192.1.0.52",
      "raw-update-file": "/home/supervisor/ipv6_nlri.bgp",
      "local-as": 3330,
      "peer-as": 3320
    }
  ]
}
}
```

1. Stop the existing blaster running instance using controller. 
2. Reboot active/standby CBNGs and load the configurations.
3. Start the existing blaster which was stopped earlier. 
4. Ensure the RD session is up & subscribers are established.
5. Check if the subscriber IP is pinging. 

**Expected behavior**

ICMP replies should be seen from the blaster for all the ICMP requests that were sent to it. 

**Screenshots**

**Additional context**

**Comment by GIC-de** [comment]
This issue is fixed in version [0.8.39](https://github.com/rtbrick/bngblaster/releases/tag/0.8.39)!

**Comment by GIC-de** [comment]
Related Commits:
+ ea8243f0000f9f96c8443d096850037a0e48cfb6
+ 7722ffe59c351b7723c9d89772ce0d4896712299
+ a353086e97331bb89570df615f04b5745e4e7920
+ cccd4fafde69ef185d45dd0f47d6e499bd604289
